build.gradle

```diff
@@ -274,10 +274,17 @@ configure(javaCodeCheckedProjects) {
     }
 
     task testSlowOnly(type: Test) {
+        dependsOn('testSlowGroovy')
         useJUnitPlatform {
-            includeEngines('junit-jupiter', 'junit-vintage', 'spock')
+            includeEngines('junit-jupiter', 'junit-vintage')
             includeTags('Slow')
         }
+    }
+
+    task testSlowGroovy(type: Test) {
```

Comment:
Had to create a custom type just for the groovy tests

Reply:
Yes, thats's what seem to be suggested as the solution, which is unfortunate: https://github.com/spockframework/spock/issues/1288#issuecomment-791901698.